<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网络调试助手</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>741</width>
     <height>611</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item row="0" column="0" colspan="2">
     <widget class="QTextEdit" name="textEditMain">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelProtocol">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>通信协议</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxProtocol">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>TCP server</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TCP client</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>UDP</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelIPAddress">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>IP地址</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxIPAddress">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>127.0.0.1</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPort">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>端口号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditPort">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>8080</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButtonOpen">
        <property name="text">
         <string>打开</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelDesAddress">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>目的地址</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyComboBox" name="comboBoxDesAdddress"/>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QTextEdit" name="textEditSend">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButtonSend">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">mycombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
